projects/commudle-admin/src/assets/styles/variables.scss

```diff
@@ -6,6 +6,25 @@ $mobile-width: 576px;
 // Fonts
 $primary-font-family: "Work Sans";
 
+// Font sizes
+$fs-12: 12px;
+$fs-13: 13px;
+$fs-14: 14px;
+$fs-15: 15px;
+$fs-16: 16px;
+$fs-17: 17px;
+$fs-18: 18px;
+$fs-19: 19px;
+$fs-20: 20px;
+$fs-21: 21px;
+$fs-22: 22px;
+$fs-23: 23px;
+$fs-24: 24px;
+$fs-25: 25px;
+$fs-26: 26px;
+$fs-27: 27px;
+$fs-28: 28px;
```

Comment:
Let's get these standardized with Apoorva, as in very limited font sizes which would apply everywhere. Keeping so many would be weird :P

Reply:
Sure!